### PR TITLE
Upgrade `certifi` package

### PR DESCRIPTION
**Pull request by yusufsheiqh**
## Context

- addresses [Certifi removing TrustCor root certificate](https://github.com/advisories/GHSA-43fp-rhv2-5gv8)
- upgrades `certifi` to version `2022.12.07`

### CODE REVIEW

File: trade_remedies_api/core/notifier.py [hunk 1]
```diff
@@ -11,7 +10,7 @@ def get_client():
     """
     Return a Notification client
     """
-    return NotificationsAPIClient(settings.GOV_NOTIFY_API_KEY)
+    return NotificationsAPIClient(os.environ["GOV_NOTIFY_API_KEY"])
```
Comment (chris-pettinga) [inline trade_remedies_api/core/notifier.py:13]: Why this change?

Comment (yusufsheiqh) [reply trade_remedies_api/core/notifier.py:14]: Resolved as discussed separately.

**Review by chris-pettinga — APPROVED** [review]
